gateway/tests/e2e/prometheus.rs

```diff
@@ -315,8 +315,6 @@ async fn test_prometheus_metrics_feedback_float() {
     let response_json = response.json::<Value>().await.unwrap();
     let inference_id = response_json.get("inference_id").unwrap().as_str().unwrap();
     let inference_id = Uuid::parse_str(inference_id).unwrap();
-    // Sleep for 1 second to allow ClickHouse to write
```

Comment:
We should keep this no? Because now we're checking `/metrics` which is a different code path.

Reply:
I do not think that's why we're sleeping here but I could be wrong. These were just added yesterday.

Reply:
just realized we sleep again after, disregard